### PR TITLE
fixed smooth scrolling and disappearance of down button

### LINKED ISSUE #3114: Smooth scrolling  & disappearance of down button
**Actual Behaviour**
* While chatting  with the bot, the smooth scrolling is not working for down navigation button.  
* Reaching at the end of chat part, the down navigation button is not disappear.

**Expected Behaviour**
* Smooth scrolling should be working for down navigation button 
* Down navigation should be disappear while it reaching at end of the chat conversation.  

**Screenshots of the issue**
captured

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by simranaggarwal1999**
Fixes #3114

Changes: Fixed smooth scrolling and disappearance of down button

Demo Link: https://pr-3114-fossasia-susi-web-chat.surge.sh/

Screenshots of the change: 
smoothscrolling

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -124,7 +125,7 @@
     "test": "npm run lint && react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "predeploy": "npm run build",
-    "deploy": "surge"
+    "deploy": "surge --project ./build --domain https://pr-3058-fossasia-susi-web-chat.surge.sh"
```
Comment (anishagg17) [inline package.json]: Please don't make such kind of changes which are not related to the issue

**Comment by chauhanparth210** [comment]
Hey @simranaggarwal1999, you don't need to change in package.json to deploy the build file to surge ,it works automatic when [travis-ci](https://travis-ci.org/) pass the code and the link  is auto generated.
 
What you have to do, remove the changes from package.json and simply make PR and add the PR number to URL and in that case your demo link must be : https://pr-3187-fossasia-susi-web-chat.surge.sh/
